tf2_client: 1.0.0-1 in 'melodic/distribution.yaml' [bloom]

peci1 opened the pull request:
Increasing version of package(s) in repository `tf2_client` to `1.0.0-2`:

- upstream repository: https://github.com/tpet/tf2_client.git
- release repository: https://gitlab.fel.cvut.cz/cras/ros-release/tf2_client
- distro file: `melodic/distribution.yaml`
- bloom version: `0.11.2`
- previous version for package: `null`

## tf2_client

```
* Adding default server launcher, with install.
* Cmake refactor, namespaced cpp file, dynamic cast to subclass.
* Fix client/server switch in c++ client.
* Passing private node handle as argument.
* C++ client single node handle, clean up, exporting library.
* Log clearing listener and buffer.
* Allow to clear the listener and buffer if they are not needed any more.
* Fix creating duration object.
* Adding cpp support, clean up of parameter names, conditional param reading.
* Passing Duration object to local Buffer instead of float.
* Initial version (untested).
* Contributors: Martin Pecka, Tomas Petricek
```

peci1 committed:
tf2_client: 0.0.0-1 in 'melodic/distribution.yaml' [bloom]

clalancette commented on melodic/distribution.yaml:
```diff
@@ -13851,6 +13851,21 @@ repositories:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://gitlab.fel.cvut.cz/cras/ros-release/tf2_client.git
+      version: 0.0.0-1
```
We don't usually like to release packages with version 0.0.0; it is pretty meaningless.  Can you please update this to have some kind of non-zero number?

peci1 replied:
I don't have write access to that package since it's from one of my former colleagues who doesn't respond to my requests to give me write access.

I could probably change the version in the release repo via patches. Would that be okay?

peci1 replied:
Okay, I've updated this PR and made the changes as release patches.

peci1 committed:
Updated tf2_client to 1.0.0-1

peci1 committed:
Updated tf2_client to 1.0.0-2

peci1 commented:
I've finally got access to the upstream repo, so I switched to standard release process and dropped the release repo changes. The license file has been added.
